### ISSUE TITLE
Extent report UI is broken, CSS/JS is not loaded

### ISSUE THREAD

**Issue by avinashkandukuri** [anshooarora/extentreports-csharp#116]
I am generating the extent reports offline for Nunit and running the test in parallel. 
a new issue is showing up, that is. extent reports are not loading with the JS, it is just generating the report with static HTML data. 

Is there any alternative to fix this issue.
